pa4/checker-pa4.ml

```diff
@@ -768,26 +778,36 @@ let main () = begin
 	(* Consolidated List of User and Base classes *)
     (* This adds the bases classes to our class inheritence mapping so that they aren't treated as
 -	undeclared classes. *)
-        let base_classes = ["Bool"; "IO"; "Int"; "String"; "Object"] in
-        let user_classes = List.map (fun ((_,cname),_,_) -> cname) ast in
+    let base_classes = ["Bool"; "IO"; "Int"; "String"; "Object"] in
+    let user_classes = List.map (fun ((_,cname),_,_) -> cname) ast in
 	let all_classes = base_classes @ user_classes in
 	let all_classes = List.sort compare all_classes in
-        let vertices = ref [] in
+    let vertices = ref [] in
 
 	(* Check of Main class, else return *)
 	if not (List.mem "Main" user_classes) then begin
 		printf "ERROR: 0: Type-Check: class Main not found\n";
 		exit 1
 	end;
 
-	(* populate parent map *)
+	(* Set up global maps *)
```

Comment:
I think changing this comment to something like "Populates the method environment with all of the methods provided by declared and inherited from super classes." That would make more sense and I think would assist in understanding the flow of the program better.